src/components/DrawerMenu.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <v-navigation-drawer v-model="localDrawer" app clipped>
+      <v-list dense>
+        <template v-for="(item, i) in items">
+          <v-row v-if="item.heading" :key="i" align="center">
```

Comment:
I'm just going to mention this here because we discussed it a few days ago. List indices should not be used for `key`s . Please use a string that is unique, either by inserting on into your list data, or by using another string in the list data. 
Right now, this list is simple enough to not need this, but if we start adding and removing items, it will definitely have issues.  When we get to the dynamic nature of this list in the future, we can change it then,

Reply:
the v-for loop is for scaffolding. Most likely, it is going to be removed because as far as I remember, it was agreed to use components instead?

Reply:
Yes, that is what I was saying. For now, the list is static, so don't worry about it. I just wanted you to be aware. Also, you should always verify boilerplate code is correct for your use. It is usually only correct for a very specific use case, if correct at all.